### ISSUE TITLE
Ordering of hijack example

### ISSUE THREAD

**Issue by benmaddison** [goldbe/draft-rpki-maxlength#2]
I find the ordering of section 3 hard to follow, and distracts attention from its purpose - which is to draw attention to the additional attack vectors enabled by "loose" ROAs.
I suggest that we reorder as follows:
- description of the sub-prefix attack with "loose" ROA
- specification of revised "strict" ROA
- explanation of why sub-prefix attack is mitigated
- explanation of why prefix attack is still possible but less likely to attract traffic

Happy to do a PR if agreed on principal.

**Comment by job** [comment]
Go for it
